Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM registry.cloudogu.com/official/base-debian:12.6-1
 
 LABEL MAINTAINER="hello@cloudogu.com" \
         NAME="official/mysql" \
-        VERSION="8.0.38-3"
+        VERSION="8.4.2-1"
```

Comment:
i will remove this within the PR - to be set via the release process 

it was need to test the upgrade process locally, because the updatescripts only start if the version of the new dogu differs